tests/performance/matrix_free_kokkos.cc

```diff
@@ -0,0 +1,331 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2023 by the deal.II authors
+ *
+ * This file is part of the deal.II library.
+ *
+ * The deal.II library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE.md at
+ * the top level directory of deal.II.
+ *
+ * ---------------------------------------------------------------------
+
+ * This test compares the MatrixFree and CUDAWrapper::MatrixFree
+ * infrastructure.
+ */
```

Comment:
Would you mind to structure the Description as follows (and adding a bit more context)?
```
  * Description:
  *
  * [A small paragraph about the test]
  *
  * Status: experimental
  */
```